cardshifter-server/src/test/java/com/cardshifter/server/main/MainServerTest.java

```diff
@@ -0,0 +1,27 @@
+package com.cardshifter.server.main;
+
+import com.cardshifter.server.model.MainServer;
+import com.cardshifter.server.model.Server;
+import org.apache.log4j.PropertyConfigurator;
+import org.junit.Test;
+
+/**
+ * Created by marc on 2017-02-07.
+ */
+public class MainServerTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void testThrowWhenNoModAreLoaded() {
```

Comment:
Shouldn't it load mods from src/test/resources? (Can't remember how it is in Cardshifter at the moment)